functests/5_latency_testing/5_latency_testing_suite_test.go

```diff
@@ -29,26 +30,64 @@ import (
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
 )
 
+//TODO get commonly used variables from one shared file that defines constants
+const filePath = "../../build/_output/bin/latency-e2e.test"
```

Comment:
You probably want a different variable name here. Maybe something like `testExecutablePath`?

Reply:
addressed in https://github.com/openshift-kni/performance-addon-operators/pull/930, Thanks!